CMakeLists.txt

```diff
@@ -256,7 +257,7 @@ elseif(CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
 
   # Enables multiprocess compiles
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /MP")
-  
+
```

Comment:
Can we avoid whitespace changes in the future.

Reply:
We'll watch out for this next time